graphql_api/types/owner/owner.py

```diff
@@ -362,3 +367,10 @@ def resolve_ai_enabled_repos(owner: Owner, info) -> List[str] | None:
         )
 
     return list(queryset.values_list("name", flat=True))
+
+
+@owner_bindable.field("activatedUserCount")
+@sync_to_async
+@require_part_of_org
+def resolve_activated_user_count(owner: Owner, info: GraphQLResolveInfo) -> int | None:
```

Comment:
Don't think this can return None

Reply:
You are correct, I figured it needed to be there for when user is not in the org, but this is not true! Pushing fix.